Add 'prompt' functionality to `abp.message` object

enisn opened the pull request:
### Description
`abp.message.prompt(...)` function is added to expose SweetAlert2 input function.

Simple usage:
```js
abp.message.prompt('Please enter a name:', (value)=> console.log('Entered name:', value));
```

**Example**

```js
let options = { title: "Name Change" ,icon: "info", inputValue: "John Doe"};
let name = await abp.message.prompt('Enter a new name to change your name', options);
if (name){
 console.log('Entered name:', name );
}
```

<img width="603" height="437" alt="image" src="https://github.com/user-attachments/assets/0e6e7e92-687f-4c98-ac26-6aff315fdb65" />


### Checklist

- [x] I fully tested it as developer / designer and created unit / integration tests
- [x] I documented it (or no need to document or I will create a separate documentation issue)

enisn committed:
Add prompt dialog support to abp.message

Introduces a new abp.message.prompt function using SweetAlert2 to display input dialogs. This allows developers to easily prompt users for text input with customizable options.

enisn committed:
Update documentation

github-actions[bot] committed:
Optimised images with calibre/image-actions

enisn committed:
Update message.md

github-actions[bot] committed:
Optimised images with calibre/image-actions

maliming reviewed (commented):
Can you also add a default implementation of `abp.message.prompt`.

https://github.com/abpframework/abp/blob/rel-10.0/modules/openiddict/app/OpenIddict.Demo.Server/wwwroot/libs/abp/core/abp.js#L301-L339

```js

const userInput = prompt("Enter a new name to change your name: ");
if (userInput !== null) {
  console.log("User enter：", userInput);
} else {
  console.log("User canceled.");
}
```

enisn committed:
Add abp.message.prompt method with warning

Introduces abp.message.prompt to display a browser prompt and invoke a callback with the result. Logs a warning indicating the method is not fully implemented.

enisn commented:
> ```js
> const userInput = prompt("Enter a new name to change your name: ");
> if (userInput !== null) {
>   console.log("User enter：", userInput);
> } else {
>   console.log("User canceled.");
> }
> ```

0661d1a091a71b6edeaebe86dc109707fbd720c1

enisn committed:
Code formatting - fix for @copilot suggestions
